rclcpp/include/rclcpp/waitable.hpp

```diff
@@ -94,7 +94,6 @@ class Waitable
   size_t
   get_number_of_ready_guard_conditions();
 
-  // TODO(jacobperron): smart pointer?
```

Comment:
I looked into this, and in my opinion it should not be a smart pointer. @jacobperron unless you disagree I'll leave this removed.